MAS Bug Fix finally complete

mas1850 opened the pull request:
I finally managed to get the project to run and fixed my chosen bug

MoralCode commented on _posts/2024-04-21-schneider-joycon.md:
```diff
@@ -0,0 +1,63 @@
+
+# Uncomment only one of the below categories
+categories:
+Bug Fix
```
```suggestion
- Bug Fix
```
Small formatting issue thats causing Jekyll to not like this file

MoralCode commented on _posts/2024-04-21-schneider-joycon.md:
```diff
@@ -0,0 +1,63 @@
+
+Here, I have many issues trying to debug the Visual Studio solution. I received a multitude of error messages, including issues with Windows SDK and .NET version differences. After meeting with Adrian, I was eventually able to launch the solution on my laptop. This is when I noticed no change within the program.
+
+![error pop-up message after editing retail_colors.xml](../assets/2024-04-21-schneider/error-message.png)
```
a couple of your image file extensions (such as this one, but also a few others throughout your post) are saved as .jpg but referenced as .png and not showing up

```suggestion
![error pop-up message after editing retail_colors.xml](../assets/2024-04-21-schneider/error-message.jpg)
```

MoralCode replied:
@mas1850 It still seems to be this specific image that isnt working

MoralCode reviewed (commented):
Looks good overall! Just a couple minor tweaks

MoralCode commented on _posts/2024-04-21-schneider-joycon.md:
```diff
@@ -0,0 +1,63 @@
+
+After fiddling with the UI some more, I found a label titled "Xenoblade Grips". Knowing that the word Xenoblade was unlikely to be used frequently, I searched the project for the word and found the correct file, named "frmJoyConColorPicker.cs". Using the foundations for color slots 17-20 and 25-28 left in the C# source code file, I added the labels and RGB codes for the eight new pastel colors.
+
+![edit to frmJoyConColorPicker.cs code](../assets/2024-04-15-schneider/code-snippet-3.jpg)
```
looks like there was just one image that slipped through
```suggestion
![edit to frmJoyConColorPicker.cs code](../assets/2024-04-21-schneider/code-snippet-3.jpg)
```

mas1850 commented:
Done

On Tue, Apr 23, 2024 at 3:27 PM Adrian Edwards ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In _posts/2024-04-21-schneider-joycon.md
> <https://github.com/FOSSRIT/hfoss2024-blogs/pull/27#discussion_r1576788088>
> :
>
> > +
> +First I made a fork of the project for me to work on. I then opened this workspace in Visual Studio with the 2017 C++ Redistributable and the Microsoft .NET Framework 4.7.1 installed, as per the mentioned prerequisites. In order to find the file I had to work on, I searched the workspace for "0AB9E6" (the Neon Blue hex code) and found the file "retail_colors.xml".
> +
> +![original retail_colors.xml code](../assets/2024-04-21-schneider/code-snippet-1.png)
> +
> +For my first draft, I made a few changes with some inconsistencies to learn the behavior of the design. My first assumption was to add more RC tags to the RetailColors and RetailColorsButtons tags. I added the four new colors to each parent tag and adjusted the numbers to increase as expected. Wondering if the preset number was the determining factor of the color's position, I changed the Pastel Purple color preset from 11 to 26.
> +
> +![first edit to retail_colors.xml code](../assets/2024-04-21-schneider/code-snippet-2.png)
> +
> +Here, I have many issues trying to debug the Visual Studio solution. I received a multitude of error messages, including issues with Windows SDK and .NET version differences. After meeting with Adrian, I was eventually able to launch the solution on my laptop. This is when I noticed no change within the program.
> +
> +![error pop-up message after editing retail_colors.xml](../assets/2024-04-21-schneider/error-message.png)
> +
> +After fiddling with the UI some more, I found a label titled "Xenoblade Grips". Knowing that the word Xenoblade was unlikely to be used frequently, I searched the project for the word and found the correct file, named "frmJoyConColorPicker.cs". Using the foundations for color slots 17-20 and 25-28 left in the C# source code file, I added the labels and RGB codes for the eight new pastel colors.
> +
> +![edit to frmJoyConColorPicker.cs code](../assets/2024-04-15-schneider/code-snippet-3.jpg)
>
> looks like there was just one image that slipped through
> ⬇️ Suggested change
>
> -![edit to frmJoyConColorPicker.cs code](../assets/2024-04-15-schneider/code-snippet-3.jpg)
> +![edit to frmJoyConColorPicker.cs code](../assets/2024-04-21-schneider/code-snippet-3.jpg)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FOSSRIT/hfoss2024-blogs/pull/27#pullrequestreview-2018125308>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7LBHCPRMZSGP4TUZYFSX53Y62YZ5AVCNFSM6AAAAABGR4UG7KVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAMJYGEZDKMZQHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
